utils/text_formatting.tsx

```diff
@@ -46,7 +46,7 @@ export type MentionKey = {
 export type Team = {
     id: string;
     name: string;
-    displayName: string;
+    display_name: string;
```

Comment:
This seems like it could even just be converted to the proper `Team` type, but opted to instead make it compatible with that type for now.